internal/storage/bucket_handle_test.go

```diff
@@ -256,3 +257,94 @@ func (t *BucketHandleTest) TestCreateObjectMethodWhenGivenGenerationObjectNotExi
 	AssertEq(nil, obj)
 	AssertTrue(strings.Contains(err.Error(), "Error 412: Precondition failed"))
 }
+
+func (t *BucketHandleTest) TestGetProjectValueWhenGcloudProjectionIsNoAcl() {
+	proj := getProjectionValue(gcs.Projection(storage.ProjectionNoACL))
+
+	AssertEq(storage.Projection(1), proj)
```

Comment:
please compare enum values so its easier to understand

AssertEq(storage.ProjectionNoAcl, proj)

Reply:
fixed